.github/workflows/CD.yml

```diff
@@ -0,0 +1,40 @@
+name: CD
+
+on:
+    push:
+      tags:
+        - '*'
+
+env:
+  PY_VERSION: "3.11"
```

Comment:
If this is something that'd you'd rather modify without opening a PR, you can make it a `vars.PY_VERSION` instead - just a thought :)

Reply:
Yeah it makes sense since we're using this in both CI and CD scripts.